ruby/ql/lib/codeql/ruby/frameworks/StandardLibrary.qll

```diff
@@ -333,6 +333,18 @@ class ModuleEvalCallCodeExecution extends CodeExecution::Range, DataFlow::CallNo
   override DataFlow::Node getCode() { result = this.getArgument(0) }
 }
 
+/**
+ * A call to `Module#const_get`, which interprets its argument as a Ruby constant.
+ * Passing user input to this method may result in instantiation of arbitrary Ruby classes.
+ */
+class ModuleConstGetCallCodeExecution extends CodeExecution::Range, DataFlow::CallNode {
+  ModuleConstGetCallCodeExecution() {
+    this.asExpr().getExpr().(UnknownMethodCall).getMethodName() = "const_get"
```

Comment:
As far as I know, going via `UnknownMethodCall` is currently the only way to identify calls to methods inherited from `Module`, `Object`, `Kernel` etc. I think to do better, we'd need to improve API graphs with subclass/inheritance knowledge so that it can determine that something like
```
module Foo
  const_get "A"
end
```
is a call to `Module#const_get`.

Reply:
OK, that makes sense.

> I think that should be quite rare, though?

Yeah, I expect so.